src/util.js

```diff
@@ -83,6 +92,9 @@ Util.getScreenHeight = function() {
 };
 
 Util.requestFullscreen = function(element) {
+  if (Util.isWebViewAndroid()) {
```

Comment:
why's this necessary? also, this code path shouldn't be accessed from iOS, right? so, we should probably avoid the initial calls to `requestFullscreen` when `!Util.isWebViewAndroid()` anyway, right?